api/app/Http/Controllers/AuthController.php

```diff
@@ -35,10 +36,11 @@ public function __construct(JWTAuth $jwtAuth)
     /**
      * Get a login token.
      *
-     * @param  Request  $request
+     * @param  Request $request
+     * @param ItemTransformerInterface $transformer
      * @return Response
      */
-    public function login(Request $request)
+    public function login(Request $request, ItemTransformerInterface $transformer)
```

Comment:
As this transformer is used in all auth response methods, should it not be passed to the constructor?